ansible_collections/arista/avd/plugins/plugin_utils/eos_validate_state_utils/avdtestbase.py

```diff
@@ -82,6 +114,23 @@ def log_skip_message(
         log_level = logging.getLevelName(logging_level.upper())
         LOGGER.log(log_level, log_msg)
 
+    def is_interface_shutdown(self, interface: dict, host: str | None = None) -> bool:
+        """
+        Check if an interface is shutdown or not, considering EOS defaults.
+
+        Args:
+            interface (str): The interface to verify.
+            host (str): Host to verify. Defaults to the host running the test.
+
+        Returns:
+            bool: Returns the interface shutdown key's value, or `interface_defaults.ethernet` if not available.
+                  Returns False if both are absent, which is the default EOS behavior.
+        """
+        host_struct_cfg = self.get_host_struct_cfg(host=host) if host else self.struct_cfg
+        if "shutdown" in interface:
+            return interface["shutdown"]
+        return get(host_struct_cfg, "interface_defaults.ethernet.shutdown", default=False)
```

Comment:
You can leverage the `default` function here https://github.com/aristanetworks/ansible-avd/blob/devel/ansible_collections/arista/avd/plugins/plugin_utils/utils/default.py

```suggestion
        return default(get(interface, "shutdown"), get(host_struct_cfg, "interface_defaults.ethernet.shutdown"), False)
```

Reply:
Nice!

Reply:
Fixed in 22836aff3e4102ab512a54b400ec10a4213cff27